edx_proctoring/templates/proctored_exam/onboarding_error.html

```diff
@@ -0,0 +1,29 @@
+{% load i18n %}
+<div class="failure sequence proctored-exam {{onboarding_status}}" data-exam-id="{{exam_id}}">
+  <h3>
+    {% blocktrans %}
+      You must complete an onboarding exam before taking this proctored exam
+    {% endblocktrans %}
+  </h3>
+  <p>
+{% if onboarding_status == 'onboarding_missing' or onboarding_status ==  'onboarding_expired' %}
+    {% blocktrans %}
+    Please complete the <a href="{{onboarding_link}}">Onboarding Exam</a> before attempting this exam.
```

Comment:
I think that we wanted to not show the onboarding link if we could not get a link to an onboarding exam.  I think having a link that redirects you to the existing page (that's the behavior I'm seeing on my devstack) is kind of confusing. Could we conditionally display the link?